requirements.txt

```
@@ -3,5 +3,6 @@ comfyui @ git+https://github.com/hiddenswitch/ComfyUI.git@ce3583ad42c024b8f060d0
 aiortc
 aiohttp
 toml
+tomli
```

Looks like `websockets`  package should also be added
```suggestion
tomli
websockets
```